moveit_commander/src/moveit_commander/planning_scene_interface.py

```diff
@@ -172,6 +160,11 @@ def attach_box(self, link, name, pose=None, size=(1, 1, 1), touch_links=[]):
             aco.touch_links = [link]
         self.__submit(aco, attach=True)
 
+    def clear(self):
+        """ Remove all objects from the planning scene """
+        self.remove_attached_object()
+        self.remove_world_object()
```

Comment:
It's a useful abstraction, but it would be nicer to expose this API as sending *a single* PlanningScene message.
It's just stupid overhead that can even show in Rviz (the attached objects might actually detach in RViz before being removed).
It looks like the asynchronous backend of the class does not even have a publisher to `planning_scene` at the moment...